server/controllers/users.js

```diff
@@ -48,13 +52,49 @@ module.exports = {
         res.status(400).send(error);
       });
   },
+
+  // login a user
+
+  login(req, res) {
+    User.findOne({
+      where: {
+        email: req.body.email
+      }
+    })
+      .then(user => {
+        if (!user) {
+          return res.status(401).send({
+            message: 'Invalid user'
+          });
+        }
+        const token = jwt.sign(
```

Comment:
Since you created the `authentication.js` file, move the signing logic to that file too.